frontend/static/js/views/PostView.js

```diff
@@ -3,13 +3,14 @@ import AbstractView from "./AbstractView.js";
 export default class extends AbstractView {
     constructor(params) {
         super(params);
+        this.postID = params.id;
```

Comment:
For consistency, could this please be updated to use camel case?

```suggestion
        this.postId = params.id;
```

Reply:
Sure. Done!